libres-core/src/appleAndJsMain/kotlin/io/github/skeptick/libres/strings/PluralRules.kt

```diff
@@ -73,7 +73,7 @@ object PluralRules {
             "uk" -> Ukrainian
             "kk" -> Kazakh
             "fr" -> French
-            else -> custom[languageCode] ?: error("Plural rule for '$languageCode' not provided")
+            else -> custom[languageCode] ?: English
```

Comment:
I would like to add some logging here, but there's no logging mechanism in the library currently.